etc/kayobe/ansible/templates/os_capacity-clouds.yml.j2

```diff
@@ -1,10 +1,13 @@
 clouds:
   openstack:
     auth:
-      auth_url: "{{ secrets_os_exporter_auth_url }}"
-      application_credential_id: "{{ secrets_os_exporter_credential_id }}"
-      application_credential_secret: "{{ secrets_os_exporter_credential_secret }}"
+      auth_url: "{{ secrets_os_capacity_auth_url }}"
+      application_credential_id: "{{ secrets_os_capacity_credential_id }}"
+      application_credential_secret: "{{ secrets_os_capacity_credential_secret }}"
     region_name: "RegionOne"
     interface: "internal"
     identity_api_version: 3
     auth_type: "v3applicationcredential"
+{% if kolla_enable_tls_internal | bool %}
```

Comment:
I think this is the wrong variable here... we should be using the external endpoint in most case, that has a valid cert.

Lets just add verify false as an available workaround via a new flag like os_capacity_openstack_verify?

We should do what the other said about binding mounting the certificate in here, and I think there must be an existing example of that somewhere :/ But I would rather we didn't block on that here.

Reply:
Implemented in [977db98](https://github.com/stackhpc/stackhpc-kayobe-config/pull/697/commits/977db9887eaa8369ad8499655601582b0783b1aa)

Reply:
Certificate verification implemented in #896